application/src/logmanager.py

```diff
@@ -11,25 +11,61 @@ def __init__(self, log_file_path, absolute_path=False):
         else:
             self.log_file_path = log_file_path
 
-    def save_breathing_time_to_today(self, breathing_time_string):
-        formatted_day_string = self.convert_breathing_times_to_strings(breathing_time_string)
-        with open(self.log_file_path, 'a+') as log_file:
-            log_file.write(formatted_day_string)
+    def ask_for_journal_entry(self):
+        # Ask the user to input a journal entry
+        confirm = False
+        while not confirm:
+            journal_entry = input("Enter Journal entry\n")
+            if len(journal_entry)==0:
+                journal_entry = 'No Entry'
+            print('Current Entry: ', journal_entry)
+            if self.ask_user():
+                confirm = True
+
+        return(journal_entry)
+
+    def ask_user(self):
+        # This function asks for either yes or no.
+        check = str(input("Is this correct? (Y/N): ")).lower().strip()
```

Comment:
`"Is this correct? (Y/n): "`, capitalize the Y not the N to imply hitting enter defaults to yes